docs/resources/file.md.erb

```diff
@@ -119,6 +119,8 @@ not
 
 Without the zero prefix for the octal value, InSpec will interpret it as the _decimal_ value 644, which is octal 1024 or `-----w-r-T`, and any test for a file that is `-rw-r--r--` will fail.
 
+#### Note: If you want to validate that a file has an upper or lower bound in `mode` see the [`more_permissive_than?`(mode)](#more_permissive_than?(mode)) matcher below.
```

Comment:
Hash marks should be used for headers. It also reads odd, like an advertisement for a feature in the documentation. I'd prefer to see this line removed.

Reply:
The thought I had was to make it easy for them to find this if when they were looking at mode it would point them to the right matcher so they were less likely to try to use `its('mode') { should cmp <= 0644 }`